aicsimageio/readers/default_reader.py

```diff
@@ -126,16 +128,12 @@ def _guess_dim_order(shape: Tuple[int]) -> str:
         return Reader._guess_dim_order(shape)
 
     @property
-    def scenes(self) -> Tuple[int]:
+    def scenes(self) -> Tuple[str]:
         # There is currently an assumption that DefaultReader will not encounter
         # files with multiple scenes. But, if we do encounter a file that DefaultReader
         # hits and a user wants scene management from that file type, we can update
         # this property then.
-        return (0,)
-
-    @property
-    def current_scene(self) -> int:
-        return self.scenes[0]
+        return ("Image:0",)
```

Comment:
this is a very ome-specific scene name -- is that intentional?

Reply:
Yea sort of kind of. Figured why not keep it consistent?